server/container_create.go

```diff
@@ -145,8 +145,13 @@ func resolveSymbolicLink(path string) (string, error) {
 }
 
 func addDevices(sb *sandbox.Sandbox, containerConfig *pb.ContainerConfig, specgen *generate.Generator) error {
+	var securityContext pb.LinuxSandboxSecurityContext
 	sp := specgen.Spec()
-	if containerConfig.GetLinux().GetSecurityContext().Privileged {
+	if securityContext := containerConfig.GetLinux().GetSecurityContext(); securityContext == nil {
+		securityContext = &pb.LinuxContainerSecurityContext{}
+	}
```

Comment:
this can be done in 3 lines:
```
securityContext := containerConfig.GetLinux().GetSecurityContext()
if securityContext == nil {
    securityContext = &pb.LinuxContainerSecurityContext{}
}
```
please change to the above.

Reply:
Fixed

Reply:
this doesn't seem fixed :confused: at least from the github diff